### ISSUE TITLE
Allow empty line after JS Doc in require-jsdoc rule

### ISSUE THREAD

**Issue by curiousity4ever** [gajus/eslint-plugin-jsdoc#323]
Currently, the following block of code returns the following exception:

```
/**
 * JS Doc rules!
 */

function ignoreEmptyLineAbove() {
    rule();
}
```

```
 5:1  error  Missing JSDoc comment                                          jsdoc/require-jsdoc
```

Is there a way to make the plugin ignore the empty line after JS Doc / before the statement?

**Comment by golopot** [comment]
It make sense to allow empty line between jsdoc and functions. Jsdoc doc generator allows it, and [projects in the wilds](https://github.com/expressjs/express/blob/4.17.1/lib/express.js) uses it.

**Comment by brettz9** [comment]
Closed by #353.
